modelplugin/aether-1.0.0/examples/set.apn-profile.gnmi

```diff
@@ -0,0 +1,75 @@
+prefix: <
+    target: 'spgw-1'
+    elem: <
+        name: 'apn-profile'
+    >
+    elem: <
+        name: 'apn-profile'
+        key:<
+            key:'id'
+            value:'_default_profile_'
```

Comment:
why the underscores in name?

Reply:
That came from the SD-Core sample JSON. Other profiles that were not the default didn't have the underscores. I just picked one at random. From Aether-config's perspective I think we can treat the name as opaque.